CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 ## [Unreleased]
 ### Added
 - Mail filtering configured via the JMAP protocol
+- JPA support for Sieve script storage
```

Comment:
you should add "(thanks to...)", that's always nice to thank contributors!

Reply:
Good habit I agree